packages/themes/pennwell/styles/components/_ads.scss

```diff
@@ -62,3 +62,53 @@
     box-shadow: 0 4px 8px 0 rgba(0, 0, 0, .2), 0 6px 20px 0 rgba(0, 0, 0, .19);
   }
 }
+
+.sticky-leaderboard {
+  &__container {
+    position: fixed;
+    right: 15px;
+    bottom: 0;
+    left: 15px;
+    z-index: 100000;
+    padding: map-get($spacers, 3);
+    text-align: center;
+    visibility: hidden;
+    background-color: $white;
+    border-radius: $border-radius;
+    border-bottom-right-radius: 0;
+    border-bottom-left-radius: 0;
+    box-shadow: $theme-box-shadow-lg;
+    opacity: 0;
+    transition: visibility linear 250ms, opacity ease 250ms;
+
+    @include media-breakpoint-up("xl", $grid-breakpoints) {
+      right: inherit;
+      left: calc(50% - 585px);
```

Comment:
The 585px and 1170px values will "break" if the xl grid breakpoint changes. This should be determined mathematically using SASS